components/image-builder-bob/pkg/builder/builder.go

```diff
@@ -83,20 +86,13 @@ func (b *Builder) buildBaseLayer(ctx context.Context, cl *client.Client) error {
 	return buildImage(ctx, b.Config.ContextDir, b.Config.Dockerfile, b.Config.WorkspaceLayerAuth, b.Config.BaseRef)
 }
 
-func (b *Builder) buildWorkspaceImage(ctx context.Context, cl *client.Client) (err error) {
+func (b *Builder) buildWorkspaceImage(ctx context.Context) (err error) {
 	log.Info("building workspace image")
 
-	contextDir, err := os.MkdirTemp("", "wsimg-*")
-	if err != nil {
-		return err
-	}
-
-	err = ioutil.WriteFile(filepath.Join(contextDir, "Dockerfile"), []byte(fmt.Sprintf("FROM %v", b.Config.BaseRef)), 0644)
-	if err != nil {
-		return xerrors.Errorf("unexpected error creating temporal directory: %w", err)
-	}
+	logs.Warn.SetOutput(os.Stderr)
+	logs.Progress.SetOutput(os.Stderr)
```

Comment:
I assume the intent here is to make it so that we stream these results back (like buildkit used to, but instead now we're just copying).

Is warn necessary? Can we do info?

Reply:
Perfect. 

Can we exclude warn?